bfdd/bfdd_nb_config.c

```diff
@@ -649,8 +843,10 @@ int bfdd_bfd_sessions_single_hop_desired_transmission_interval_modify(
 
 	case NB_EV_APPLY:
 		bs = nb_running_get_entry(args->dnode, NULL, true);
-		bs->peer_profile.min_tx =
-			yang_dnode_get_uint32(args->dnode, NULL);
+		if (tx_interval == bs->timers.desired_min_tx)
+			return NB_OK;
+
+		bs->peer_profile.min_tx = tx_interval;
 		bfd_session_apply(bs);
```

Comment:
ditto: separate commit for changes related to common bfd.

Reply:
you seem to have removed those changes.
if the behavior can be kept unchanged, it is ok to me.